cmd/dex/config_test.go

```diff
@@ -25,6 +25,7 @@ func TestValidConfiguration(t *testing.T) {
 				File: "examples/dex.db",
 			},
 		},
+		OIDCGroupsPrefix: true,
```

Comment:
This just ensures that config is valid with OIDCGroupsPrefix enabled

Reply:
:+1:

Reply:
Do we receive any kind of notification when the config is not valid somehow ?

Reply:
If we pull changes to the config from upstream that don't agree with our `OIDCGroupPrefix` parameter, this test will fail in CIrcleCI.